wire-ios-request-strategy/Sources/Helpers/ZMSimpleListRequestPaginator.m

```diff
@@ -104,6 +104,22 @@ - (ZMTransportRequest *)requestForSingleRequestSync:(ZMSingleRequestSync * __unu
     components.queryItems = queryItems;
 
     ZMTransportRequest *request = [ZMTransportRequest requestGetFromPath:components.string apiVersion:apiVersion];
+
+    ZM_WEAK(self);
+    [request addCompletionHandler:[ZMCompletionHandler handlerOnGroupQueue:self.moc block:^(ZMTransportResponse * response) {
+        ZM_STRONG(self);
```

Comment:
Is it really needed to use a custom define on this expressions?
Probably we could just use simply `__weak MyObject *weakSelf = self;`?

Reply:
this is the old macro that helps do that exactly

Reply:
Okay, I prefer do have the "native" code, because one doesn't need to know more details or definitions. But if you use `ZM_WEAK` in other cases, we can keep it for consistency.